SyncDocs/SyncDocs.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <UserSecretsId>1630f78f-83d6-46aa-8b27-620733e6df2f</UserSecretsId>
```

Comment:
Haven't looked at the details but any specific reason the project has a usersecretid?

Reply:
The DeepL secrect I stored locally.